### ISSUE TITLE
A lot of timeouts!

### ISSUE THREAD

**Issue by zucram** [gjohansson-ST/sector#118]
**Describe the bug**
Not sure if there’s much to be done but the integration seems to time out a lot. 74 times today (refresh rate of 60) which leads to the lock and panel being unavailible. See attached screenshot. 
![IMG_0308](https://user-images.githubusercontent.com/5426869/193603098-4e778109-52f7-4149-b6d8-6e2021a042a3.jpeg)

**Version**
Latest beta

**Comment by gjohansson-ST** [comment]
Yes. Unfortenately sector's api is highly unreliable and does a lot of timeout.
I will change this "unavailable" behavior given this so the sensors etc. doesn't go unavailable and instead perhaps implement a sensor where it's visible.

**Comment by chribster** [comment]
@gjohansson-ST - I did not notice the resulting "unavailable" behaviour in the 0.3.4 version.

Is that a consequence of the upgraded version?

I downgraded to 0.3.4 since the devices becoming unavailable messed with my "is the door locked when leaving" behaviour

**Comment by gjohansson-ST** [comment]
I will fix that so the entities remain available even at timeouts to not mess with a bad performing api (which is the case here).
Will come a new version shortly.

**Comment by gjohansson-ST** [comment]
Let me know how this works with the latest version https://github.com/gjohansson-ST/sector/releases/tag/v0.3.6

**Comment by jlatvala** [comment]
Not sure if this is the same, but I am getting these now in 0.3.6:
(3 different log entries, all below)

```
Logger: custom_components.sector
Source: custom_components/sector/coordinator.py:**233**
Integration: Sector Alarm ([documentation](https://github.com/gjohansson-ST/sector/blob/master/readme.md), [issues](https://github.com/gjohansson-ST/sector/issues))
First occurred: 08:16:36 (**10 occurrences**)
Last logged: 12:57:52

Timeout on during login
```


```
Logger: custom_components.sector
Source: custom_components/sector/coordinator.py:**279**
Integration: Sector Alarm ([documentation](https://github.com/gjohansson-ST/sector/blob/master/readme.md), [issues](https://github.com/gjohansson-ST/sector/issues))
First occurred: 10:02:37 (**2 occurrences**)
Last logged: 12:34:12

Timeout on during login
```


This happened right after HA reboot:
```
Logger: custom_components.sector
Source: custom_components/sector/coordinator.py:**133**
Integration: Sector Alarm ([documentation](https://github.com/gjohansson-ST/sector/blob/master/readme.md), [issues](https://github.com/gjohansson-ST/sector/issues))
First occurred: 08:03:25 (1 occurrences)
Last logged: 08:03:25

Timeout on during login
```

**Comment by gjohansson-ST** [comment]
Yes. I changed so it's now printing a warning on timeout instead of raising an error hence it doesn't destroy the integration but instead just writes it in the log.
So it's on purpose (at the moment)

**Comment by chribster** [comment]
@gjohansson-ST - 0.3.6 has not made my entities unavailable, so the fix seems to be working just fine 👍🏼

**Comment by gjohansson-ST** [comment]
Fixed with 0.3.7
